modules/data.atmosphere/R/GEFS_helper_functions.R

```diff
@@ -0,0 +1,906 @@
+#' Download gridded forecast in the box bounded by the latitude and longitude list
+#'
+#' @param lat_list
+#' @param lon_list
+#' @param forecast_time
+#' @param forecast_date
+#' @param model_name_raw
+#' @param num_cores
+#' @param output_directory
+#'
+#' @return
+#' @export
```

Comment:
Are these helpers intended for use outside the data.atmosphere package, or are they just to support the download.NOAA_GEFS function? If the latter, I suggest leaving them unexported.

Reply:
I removed the export for the GEFS specific functions! And I consolidated the downscale functions that we had previously for GEFS into one file in the last push I made. I still have those as exported because they could be used to downscale other met data but I can remove that if you think it would be better not being exported

Reply:
I always err on the side of exporting more, not less. The big disadvantage of exporting, in my mind, is that it potentially causes conflicts of multiple functions with the same name

Reply:
> I still have those as exported because they could be used to downscale other met data

👍 👍 That seems exactly right to me. Thanks!